packages/components/src/mobile/bottom-sheet/button.native.js

```diff
@@ -8,14 +8,28 @@ import { TouchableOpacity, View, Text } from 'react-native';
  */
 import styles from './styles.scss';
 
-const BottomSheetButton = ( { onPress, disabled, text, color } ) => (
+const BottomSheetButton = ( {
+	onPress,
+	disabled,
+	text,
+	color,
+	justifyContent,
```

Comment:
I don't think we need these changes after the latest update right?

Reply:
You are right @geriux. I removed the unneeded BottomSheetButton customisation.